### DISCUSSION TITLE
Can only use channelId and not tread

### DISCUSSION THREAD

**Discussion by ghost** [thejoabo/virtualfisher-bot#89] [Ideas]
hi, I'm in a server with multiple friend and we all fish in tread on a channel bc of the fish spam and was wondering if it would be possible to add tread support to the bot bc rn we can only use the autobot in channel not tread

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/49594896/219922224-86221123-69c1-458e-99ef-7d49472250ff.png)

**Comment by thejoabo** [comment]
Hello @ thats a interesting idea, I could add a feature to automatically create a tread on the selected channel to avoid problems related to messages being addressed to the wrong user (in case of multiple people using the bot/fishing in the same channel). I'll investigate further that possibility on future updates, thanks for your suggestion !

**Comment by Maxhem2** [comment]
Would be great since the bot just told me in the cmd overview I found and artifact and more while it was a different user that fished them
